python/mlc_chat/compiler/model/__init__.py

```diff
@@ -1,2 +1,3 @@
 """Model definition for the compiler."""
+from . import llama, mistral
```

Comment:
@junrushao let me know if this structure makes more sense. Each model implements its own model/loader/quantization (without dependencies), so that any change does not affect any model

Reply:
This is great! Thanks for bringing in this change 🙏